xlators/mgmt/glusterd/src/glusterd-volgen.c

```diff
@@ -2595,23 +2595,23 @@ brick_graph_add_pump(volgen_graph_t *graph, glusterd_volinfo_t *volinfo,
  * the topology of the brick graph */
 static volgen_brick_xlator_t server_graph_table[] = {
     {brick_graph_add_server, NULL},
```

Comment:
Note: I have not added the dbg_key string alongside `brick_graph_add_server` in the above line. I saw that even if I did, it seemed to fail for some other reason.  I did not investigate it further because I felt that custom xlator should not be added above server or below posix. i.e. it always has to be in between these 2 terminal xlators. What do you think?

Reply:
I hope this point is okay too.